### ISSUE TITLE
Pokemon Go : Make a Page showing all released Unowns

### ISSUE THREAD

**Issue by dyreschlock** [dyreschlock/schlock-website#34]
Make an "app" page that shows all the currently released Unowns.

This app page should list all of the events that have Unowns and which unknowns were available.  Also the Date of the event, and the region in the world.  Color code that, too.

With the same information, the app should reorder the list sorted by each Unown, showing all the events they were available.

Also, a final list sorted by rarity depending on the number of times each Unown was available.

Basically, do this page: http://ark42.com/pogo/unown.php

After, as an added feature, keep track of the Unown each account has.

**Comment by dyreschlock** [comment]
I think this is done.  Additional ideas and features can have their own cases.

The page can be found at /apps/pokemon/unown

There are 4 lists: custom list, list by rarity, list by letter, and list of events.  The custom list is meant to show which Unown have not been obtained, but I'll open up another case to make this inputable to allow people to type in the ones they do have.
